flatpak/manifest_tool/manifest_ops.py

```
@@ -1,382 +1,16 @@
-        modules.insert(insert_index, module_name)
-
-    document.mark_changed()
-    _LOGGER.debug("Ensured module include %s", module_name)
-    return OperationResult.changed_result(f"Ensured module include {module_name}")
+"""Manifest operations module - backward compatibility wrapper.
+
+This module imports all functions from the new modular operations package
+to maintain backward compatibility with existing code.
+"""
+
+from .operations.manifest import *
```

Ruff (0.13.1)

`from .operations.manifest import *` used; unable to detect undefined names

(F403)